src/main/scala/com/hs/pipeline/demo/transformations/ContractTransformation.scala

```diff
@@ -14,4 +16,13 @@ object ContractTransformation {
     contractWithCountryCols
   }
 
+  def calParticipation(contractDf: DataFrame): DataFrame = {
```

Comment:
rename the function to a more significant name like calculateParticipation or addParticipationColumn